### DISCUSSION TITLE
Domain Crud Apis

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3185] [Development discussions]
Hello,

How can I use the domains crud api created structure? Do I need to add these files to my Postal server?

https://github.com/postalserver/postal/pull/992/commits/b03e2a8b7cba1bc2485d66ad6263a995d6bf483a#diff-e3377800ed798dcf052a5f4f6a864e5f2e1d70e68f64bb9740c0ab2323e4a716

**Comment by willpower232** [comment]
There isn't an official domain crud API currently, if you add those files to your docker images then they may work but I can't say for certain and they're definitely unsupported officially.

**Reply by TR-SametOzen** [reply]
How can I add these?
